### PR TITLE
MF-503 - Update readme

### PR DESCRIPTION
**Pull request by dashton82**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,91 @@
...
+These functions deal with changing the state of reservation information and consume the following nServiceBus events:
+
+- DraftApprenticeshipCreatedEvent
+- DraftApprenticeshipDeletedEvent
+- ReservationCreatedEvent
+- ReservationDeletedEvent
+
+The first two DraftApprenticeship Events come from the Approvals/Commitments solultion. The CreatedEvent marks a reservation as used so a further apprenticeship cannot be assigned to it. The DeletedEvent frees that reservation up to be used against another apprenticeship. The two Reservation events are triggers for the elastic index to be updated with those reservations for the providers that have permissions. 
+
+Also off the back of any reservation created or deleted by a provider, if that employer has there notification settings set to receive emails, then they will get an email informing them that someone has deleted or created a reservation on their behalf.
```
Comment (robinoconnor) [inline README.md]: there -> their

File: README.md [hunk 2]
```diff
@@ -1 +1,91 @@
...
+
+### SFA.DAS.Reservations.Functions.ProviderPermission
+This function is responsible for consuming events from ProviderPermissions. It handles the following nServiceBus events:
+
+- UpdatedPermissionsEvent
+
+Provider permission information is stored for use of building the elastic search index used for providers. When an event is received we add or remove the employers reservations for that provider to be able to search against.
+
+### SFA.DAS.Reservations.Functions.RefreshCourse
+This is a timed job, runs at midnight and refreshes all Standards into the `Course` table pulling data from the Apprenticeship Info Service API. It can also be manually invoked via a HTTP POST request.
```
Comment (robinoconnor) [inline README.md]: Just checking we don't use  frameworks as well?

File: README.md [hunk 3]
```diff
@@ -1 +1,91 @@
...
+- Clone repository
+- Set EnvironmentName to **DEV** in local.settings.json
+
+All chances will now happen in the in memory db context. Further Azure function capabilities are described below.
+
+You are also able to run in **LOCAL** mode, for this you need the following dependencies:
+
+- Docker
+- SQL Server, can be hosted on docker - the database that is part of the [das-reservations-api](https://github.com/SkillsFundingAgency/das-reservations-api) should be published
+- Elastic running on docker
```
Comment (robinoconnor) [inline README.md]: Elastic Search

File: README.md [hunk 4]
```diff
@@ -1 +1,91 @@
...
\ No newline at end of file
+# das-reservations-jobs
+
+## Build Status
+
+![Build Status](https://sfa-gov-uk.visualstudio.com/Digital%20Apprenticeship%20Service/_apis/build/status/Manage%20Funding/das-reservations-jobs?branchName=master)
+
+## Requirements
+
+DotNet Core 2.2 and any supported IDE for DEV running. Azure function SDK v2 is also required.
```
Comment (robinoconnor) [inline README.md]: Just remember that 2.2 is not LTS version so 2.1 or 3.1 would be better

File: README.md (hunk 2 above)
Comment (dashton82) [reply README.md]: yeah no frameworks

File: README.md (hunk 3 above)
Comment (dashton82) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (dashton82) [reply README.md]: Agreed - although all the others are 2.2 🤦‍♂ might be worth upgrading to 3 at some point

File: README.md (hunk 2 above)
Comment (dashton82) [reply README.md]: nah its just standards

File: README.md (hunk 4 above)
Comment (robinoconnor) [reply README.md]: ok i'll leave that then as a separate task

**Pull request merged**